README.md

```diff
@@ -1 +1,2 @@
-# admission-trial-code-review
\ No newline at end of file
+# HTML Project
+ Microverse pre-enrollment bootcamp HTML project
```

Comment:
You README file has a description of the project. This is good. However please update  it by adding more relevant sections. You can use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. ✌🏻